frontend/src/components/ModalPortal.tsx

```diff
@@ -0,0 +1,12 @@
+import { ReactNode } from 'react';
+import ReactDom from 'react-dom';
+
+interface Props {
+  children?: ReactNode;
+}
+function ModalPortal({ children }: Props) {
+  const el = document.getElementById('modal');
```

Comment:
`Portal` 같은 경우에는 `Modal` 이 아닌 다른 역할로도 사용할 수 있으니깐 그냥 `Portal` 로 지정하고 인터페이스로 `portalId` 같은걸 받아서 좀 더 범용적으로 사용하도록 하는건 어떨까?

아래는 예시

```suggestion

interface Props {
   children?: ReactNode;
   portalId: string;
 }
function Portal({ children, portalId }: Props) {
  const el = document.getElementById(portalId);
```